ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+// import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: '#FAFAFA',
+    height: 200
+  },
+  centered: {
+    width: '100%'
+  },
+  title: {
+    fontSize: 14
+  }
+
+}))
+
+function FeatureCard (props: FeatureCardProps): JSX.Element {
+  const classes = useStyles()
+
+  return (
+    <Card className={`${classes.root}`} variant="outlined">
+        <CardContent>
+            <Grid container>
```

Comment:
<img width="941" alt="Screen Shot 2021-03-17 at 4 04 05 PM" src="https://user-images.githubusercontent.com/35741256/111531604-20520b00-873b-11eb-9435-0a29e5a97b81.png">

Seems like maybe there's too much empty space beneath the description. Not exactly sure what's causing that. Also, these particular icons are pretty small, maybe they need to be `fontSize='large'` like the others?

Reply:
Yeah the font size is wrong, i forgot to change it from the original smaller boxes.

I'm not particularly focused on the fine details of how the css looks right now, but I can make them smaller..  I feel like we might be focusing too much on details though